docs/install_shadow.md

```diff
@@ -36,14 +36,15 @@ After running `./setup install`, you can find the list of installed files in
 
 ## Setup Notes
 
-  + All build output is generated out-of-source, by default to the `./build`
-    directory.
+  + All build output is generated out-of-source to the `./build` directory.
```

Comment:
This parses slightly ambiguously: "out-of-source-to" vs "out-of-source; to". The former is nonsensical, but the ambiguity makes it a little awkward.

Maybe just "All build output is generated to the `./build` directory"?